src/makeRobotAccountant.js

```diff
@@ -6,7 +6,13 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x) =>
+    // eslint-disable-next-line no-sequences
+    (++count, (y) => (count > 3 && count % 2 === 0)
+      ? 'Bzzz... Error!'
+      : x + y);
 }
```

Comment:
```suggestion
  return (x) => {
    ++count; 
    return (y) => {
      count > 3 && count % 2 === 0
        ? 'Bzzz... Error!'
        : x + y
    }
}
```
Let's not disable es-lint

Reply:
Спасибо